packages/server/tests/clients/servicesClient.ts

```diff
@@ -0,0 +1,246 @@
+        const contractId = contractReceipt?.contractId;
+
+        this.logger.info(`new contract ID: ${contractId?.toString()}`);
+
+        return contractId;
+    };
+
+    async executeContractCall(contractId) {
+        // Call a method on a contract exists on Hedera, but is allowed to mutate the contract state
+        this.logger.info(`Execute contracts ${contractId}'s createChild method`);
+        const contractExecTransactionResponse =
+            await this.executeTransaction(new ContractExecuteTransaction()
+                .setContractId(contractId)
+                .setGas(75000)
+                .setFunction(
+                    'createChild',
+                    new ContractFunctionParameters()
+                        .addUint256(1)
+                ));
+
+        // @ts-ignore
+        const resp = await this.getRecordResponseDetails(contractExecTransactionResponse);
+        const contractExecuteTimestamp = resp.executedTimestamp;
+        const contractExecutedTransactionId = resp.executedTransactionId;
+
+        return { contractExecuteTimestamp, contractExecutedTransactionId };
+    };
+
+    async createAliasAccount(initialBalance = 1000) {
+        const privateKey = PrivateKey.generateECDSA();
+        const publicKey = privateKey.publicKey;
+        const aliasAccountId = publicKey.toAccountId(0, 0);
+
+        this.logger.trace(`New Eth compatible privateKey: ${privateKey}`);
+        this.logger.trace(`New Eth compatible publicKey: ${publicKey}`);
+        this.logger.debug(`New Eth compatible account ID: ${aliasAccountId.toString()}`);
+
+        const aliasCreationResponse = await this.executeTransaction(new TransferTransaction()
+            .addHbarTransfer(this._thisAccountId(), new Hbar(initialBalance).negated())
+            .addHbarTransfer(aliasAccountId, new Hbar(initialBalance)));
+
+        this.logger.debug(`Get ${aliasAccountId.toString()} receipt`);
+        await aliasCreationResponse?.getReceipt(this.client);
+
+        const balance = await this.executeQuery(new AccountBalanceQuery()
+            .setAccountId(aliasAccountId));
+        this.logger.info(`Balance of the new account: ${balance.toString()}`);
+
+        const accountInfo = await this.executeQuery(new AccountInfoQuery()
+            .setAccountId(aliasAccountId));
+        this.logger.info(`New account Info: ${accountInfo.accountId.toString()}`);
+        return {
+            alias: aliasAccountId,
+            accountId: accountInfo.accountId,
+            address: accountInfo.contractAccountId,
+            client: new ServicesClient(
+                this.network,
+                accountInfo.accountId.toString(),
+                privateKey.toString(),
+                this.logger.child({ name: `services-client` })
+            ),
+            wallet: new ethers.Wallet(privateKey.toStringRaw())
+        };
+
+    };
+
+    _thisAccountId() {
+        return this.client.operatorAccountId || AccountId.fromString('0.0.0');
```

Comment:
WHen is the fall back ever used. Shouldn't all tests fail if the client isn't configured right on startup?

Reply:
Practically `this.client.operatorAccountId` will/should never be `unidentified`, however the SDK implementation returns `AccountId || null` for `.operatorAccountId` so the IDE marks it as an error and says that the operator account ID is potentially `null` so it **should not** be used within other functions f.e adding it as a transfer sender when doing HBAR transfer. 

I've added `|| AccountId.fromString('0.0.0')` just to remove the error from the IDE that was all over the place